### PR TITLE
Checkstyle

### PR DESCRIPTION
**Pull request by rodionovsasha**
Added custom checkstyle support for the build.
Default config file https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/sun_checks.xml

### CODE REVIEW

File: src/main/java/com/github/vkorobkov/jfixtures/JFixtures.java [hunk 1]
```diff
@@ -6,7 +6,10 @@
 import com.github.vkorobkov.jfixtures.sql.dialects.MySql;
 import com.github.vkorobkov.jfixtures.sql.dialects.PgSql;
 
-public class JFixtures {
+public final class JFixtures {
+    private JFixtures() {
```
Comment (vkorobkov) [inline src/main/java/com/github/vkorobkov/jfixtures/JFixtures.java:10]: Use `lombok` for constructor generation?

File: src/main/java/com/github/vkorobkov/jfixtures/instructions/CleanTable.java [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package com.github.vkorobkov.jfixtures.instructions;
 
+import lombok.Getter;
+
+@Getter
 public class CleanTable implements Instruction {
-    public final String table;
+    private final String table;
 
     public CleanTable(String table) {
```
Comment (vkorobkov) [inline src/main/java/com/github/vkorobkov/jfixtures/instructions/CleanTable.java]: Can we generate constructor with lombok?

Comment (vkorobkov) [inline src/main/java/com/github/vkorobkov/jfixtures/instructions/CleanTable.java]: Can we generate constructor with lombok?

File: src/main/java/com/github/vkorobkov/jfixtures/util/SqlUtil.java [hunk 3]
```diff
@@ -2,20 +2,22 @@
 
 import java.util.Objects;
 
-public class SqlUtil {
+public final class SqlUtil {
     private static final String SINGLE_QUOTE = "'";
     private static final String SINGLE_QUOTE_ESCAPED = "''";
 
+    private SqlUtil() {
```
Comment (vkorobkov) [inline src/main/java/com/github/vkorobkov/jfixtures/util/SqlUtil.java:9]: Use lombok for the constructor generation?

File: src/main/java/com/github/vkorobkov/jfixtures/util/StreamUtil.java [hunk 4]
```diff
@@ -2,8 +2,13 @@
 
 import java.util.function.BinaryOperator;
 
-public class StreamUtil {
+public final class StreamUtil {
+    private StreamUtil() {
```
Comment (vkorobkov) [inline src/main/java/com/github/vkorobkov/jfixtures/util/StreamUtil.java:6]: Use lombok for the constructor generation?

**Review by vkorobkov — CHANGES_REQUESTED** [review]
please read through comments. Minor changes are required

File: src/main/java/com/github/vkorobkov/jfixtures/JFixtures.java (hunk 1 above)
Comment (rodionovsasha) [reply src/main/java/com/github/vkorobkov/jfixtures/JFixtures.java:10]: not supported
https://github.com/checkstyle/checkstyle/issues/2579

File: src/main/java/com/github/vkorobkov/jfixtures/instructions/CleanTable.java (hunk 2 above)
Comment (rodionovsasha) [reply src/main/java/com/github/vkorobkov/jfixtures/instructions/CleanTable.java]: done

File: src/main/java/com/github/vkorobkov/jfixtures/util/SqlUtil.java (hunk 3 above)
Comment (rodionovsasha) [reply src/main/java/com/github/vkorobkov/jfixtures/util/SqlUtil.java:9]: not supported

File: src/main/java/com/github/vkorobkov/jfixtures/util/StreamUtil.java (hunk 4 above)
Comment (rodionovsasha) [reply src/main/java/com/github/vkorobkov/jfixtures/util/StreamUtil.java:6]: not supported

**Review by vkorobkov — APPROVED** [review]

**Pull request merged**